client/src/components/HabitMarker.tsx

```diff
@@ -0,0 +1,28 @@
+import { FaCheck } from 'react-icons/fa';
+import { convertColor } from '../lib';
+
+type Props = {
+  mark: boolean;
+  color: string;
+};
+export function HabitMarker({ mark, color }: Props) {
+  let buttonStyle = `rounded-full border-[5px]
+    w-[40px] h-[40px] cursor-pointer
+    flex justify-center items-center border-[${convertColor(color)}]`;
+
+  if (!mark) {
+    buttonStyle += ' big:hover:bg-[#64646420]';
+  } else {
+    buttonStyle += ` bg-[${convertColor(color)}]`;
```

Comment:
I believe this style of creating your CSS will fail with Tailwind in deployment. You need to have the complete string `bg-[#FF5A5A]`, and so forth, in your code. That is, you cannot create the classname dynamically. So you'll need to change `convertColor` so that it returns the entire `bg-[#FF5A5A]` string. If you need `convertColor` for other purposes, you'll need to have two versions of the function.

I believe Dan had this problem and Shawn helped him diagnose it. You can check with them if you have trouble.

I believe there is a way to tell Tailwind which `bg-[xxx]` codes you will be using so that it bundles them into your product. That way you won't need to change your code. I'm not sure what that mechanism is, though. You'll have to research it if you want to go this route.